hack/update-codegen.sh

```diff
@@ -744,24 +844,286 @@ function codegen::informers() {
     fi
 }
 
+# $1: output package base (in Go pkg syntax)
+# $2: input package base (in Go pkg syntax)
```

Comment:
are these positional ($1 and $2) or flags (`--output_pkg_root` and `--input_pkg_root`)?

Reply:
forgot to remove those - from previous drafts